java-lotto/src/main/java/domain/Operator.java

```diff
@@ -0,0 +1,41 @@
+package domain;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum Operator {
+
+    ADD("+", (a, b) -> a + b),
+
+    SUBTRACT("-", (a, b) -> a - b),
+
+    MULTIPLY("*", (a, b) -> a * b),
+
+    DIVIDE("/", (a, b) -> {
+        if (b == 0)
+        {
+            throw new ArithmeticException("0으로 나눌 수 없습니다.");
+        }
+        return a / b;
+    });
+
+    private final String symbol;
+    private final BiFunction<Integer, Integer, Integer> operate;
+
+    Operator(String symbol, BiFunction<Integer, Integer, Integer> operate) {
+        this.symbol = symbol;
+        this.operate = operate;
+    }
+
+    public int apply(int first, int second) {
```

Comment:
BiFunction 메소드 이름을 그대로 사용하기 보다는 `Operator` 특징에 맞는 네이밍을 사용하면 어떨까요?

```suggestion
    public int operate(int first, int second) {
```